idaes/generic_models/properties/core/generic/generic_property.py

```diff
@@ -1742,7 +1785,7 @@ def calculate_scaling_factors(self):
             self.pressure, default=1, warning=True)
         sf_mf = {}
         for i, v in self.mole_frac_phase_comp.items():
-            sf_mf[i] = iscale.get_scaling_factor(v, default=1e3, warning=True)
+            sf_mf[i] = iscale.get_scaling_factor(v, default=10, warning=True)
```

Comment:
I am not completely sure about this, a we use 1e3 as the default scaling factor for mole fractions everywhere else (for better or worse). Then again, users really should be providing their own scaling in a lot of cases.

Reply:
I got this value from @eslickj when I asked him. It doesn't matter much to me either way.